learn/getting-started/building-a-data-service-in-ballerina.md

```diff
@@ -270,7 +270,7 @@ service /employees on new http:Listener(8080) {
 }
 ```
 
-This resource function may be invoked by sending a `POST` request to `http://localhost:8080/employees` with the 
+Invoke defined resource function by sending a `POST` request to `http://localhost:8080/employees` with the 
```

Comment:
```suggestion
Invoke the defined resource function by sending a `POST` request to `http://localhost:8080/employees` with the 
```

Reply:
Updated in 7e73c5565f0163ca5e5ae9de0616e88c14954143